tools/flaky-test-reporter/github_issue_test.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	"github.com/google/go-github/github"
 	"github.com/knative/test-infra/shared/ghutil/fakeghutil"
+	ftrConfig "github.com/knative/test-infra/tools/flaky-test-reporter/config"
```

Comment:
do we need an alias here? We can just use config. ftrConfig is confusing

Reply:
I felt naming this package as `config` probably is a bad idea. But naming things is so hard, tried

Reply:
alias removed